STM32Cube/Tasks/sdmmc.c

```diff
@@ -0,0 +1,187 @@
+#include "sdmmc.h"
+#include <stdio.h>
+
+UART_HandleTypeDef *sdmmcHuart;
```

Comment:
this should be an `extern` for consistency

Reply:
i.e. you should extern the handle specifically, and remove both the pointer defined here and the passing of the uart as an argument to the task